content/en/docs/instrumentation/python/automatic.md

```diff
@@ -88,6 +92,15 @@ The examples that follow send instrumentation results to the console. Learn more
 about installing and configuring the [OpenTelemetry Distro](../distro) to send
 telemetry to other destinations, like an OpenTelemetry Collector.
 
+**Note**: If you were to use auto-instrumentation and manually instrumenting your code the
+pacakage `opentelemetry-distro` won't let you set a `Resource` with attributes defined in your code, 
+you will need to set all your desired parameters through [environment variables][env]
+
+It is also possible to use the instrumentation libraries (such as `opentelemetry-instrumentation-flask`)
+by themselves which may have an advantage of customizing options. However, by choosing
+to do this it means you forego using auto-instrumentation by starting your application with
```

Comment:
We're a little more loose with the term autoinstrumentation here. It's arguable that using this library is also a form of automatic instrumentation, since it automatically instruments requests coming into your flask server.

Reply:
I could go with, `auto-instrumentation through libraries`, `auto-instrumentation through sdk / cli command` and `manual instrumentation`. Where I would define these terms high up in this doc.